### PR TITLE
Add ReplicaNumber option in LoadCollection

### PR DESCRIPTION
**Pull request by letian-jiang**
Add ReplicaNumber option in LoadCollection

Related to #272 

Signed-off-by: Letian Jiang <letian.jiang@outlook.com>

### CODE REVIEW

File: client/client_grpc_options.go [hunk 1]
```diff
@@ -16,22 +16,20 @@ import (
...
 )
 
-// CreateCollectionOpt is option interface for CreateCollection
-type CreateCollectionOpt interface {
-	OptCreateCollection(req *server.CreateCollectionRequest)
-}
-
-// createCollectionOptFunc shortcut function type
-type createCollectionOptFunc func(*server.CreateCollectionRequest)
+type CreateCollectionOption func(*server.CreateCollectionRequest)
```
Comment (congqixia) [inline client/client_grpc_options.go:19]: Need comment here

Comment (letian-jiang) [reply client/client_grpc_options.go:20]: Fixed

File: client/client_grpc_options_test.go [hunk 2]
```diff
@@ -13,59 +13,57 @@ package client
...
+	mock.setInjection(mHasCollection, hasCollectionDefault)
+	defer mock.delInjection(mHasCollection)
+	t.Run("Load default replica", func(t *testing.T) {
+		mock.setInjection(mLoadCollection, func(ctx context.Context, raw proto.Message) (proto.Message, error) {
+			req, ok := raw.(*server.LoadCollectionRequest)
+			if !ok {
+				return badRequestStatus()
+			}
+			assert.Equal(t, testDefaultReplicaNumber, req.GetReplicaNumber())
+			assert.Equal(t, testCollectionName, req.GetCollectionName())
```
Comment (congqixia) [inline client/client_grpc_options_test.go:48]: This test case shall be in client_grpc_collection_test.go

Comment (congqixia) [reply client/client_grpc_options_test.go:48]: My bad to leave it here. Could you please move it to the correct file?

Comment (letian-jiang) [reply client/client_grpc_options_test.go]: Done. Please check again.

File: client/client_grpc_options_test.go [hunk 3]
```diff
@@ -12,60 +12,33 @@
...
 
-type mockCreateCollectionOpt struct {
-	tmock.Mock
-}
-
-func (m *mockCreateCollectionOpt) OptCreateCollection(req *server.CreateCollectionRequest) {
-	m.Called(req)
-}
-
-func TestCreateCollection_WithConsistencyLevel(t *testing.T) {
```
Comment (congqixia) [inline client/client_grpc_options_test.go:47]: Is this test case removed?

Comment (letian-jiang) [reply client/client_grpc_options_test.go:47]: Have added.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**